src/sentry/monitors/models.py

```diff
@@ -324,7 +324,16 @@ def get_schedule_type_display(self):
         return ScheduleType.get_name(self.config["schedule_type"])
 
     def get_audit_log_data(self):
-        return {"name": self.name, "type": self.type, "status": self.status, "config": self.config}
+        return {
+            "name": self.name,
+            "type": self.type,
+            "status": self.status,
+            "config": self.config,
+            "is_muted": self.is_muted,
+            "slug": self.slug,
+            "owner_user_id": self.owner_user_id,
+            "owner_team_id": self.owner_team_id,
```

Comment:
Maybe instead we can use the `owner_actor` field https://github.com/getsentry/sentry/blob/7549cbdaf78c520047f4540e3207074d5bf86f7b/src/sentry/monitors/models.py#L303-L305

Then we can call [`resolve`](https://github.com/getsentry/sentry/blob/7549cbdaf78c520047f4540e3207074d5bf86f7b/src/sentry/utils/actor.py#L99-L100) and use the string repr. Though idk what we typically do for audit logs cc @wedamija